test/integration/setup.js

```diff
@@ -22,31 +22,28 @@ const request = require('request');
 
 // Run before all tests
 before(function(done) {
-	const that = this;
-
-	this.baseUrl = 'http://localhost:' + config.port + '/';
+	this.baseUrl = `http://localhost:${config.port}/`;
 	this.app = null;
 	this.last = {};
-	this.navigate = createNavigator(that.baseUrl, that.last);
+	this.navigate = createNavigator(this.baseUrl, this.last);
```

Comment:
Why are we removing `that` here? Is this a previous bug that we're fixing?

Reply:
It was missed previously when switching from `that → this`